documents/src/pages/elements/rating.md

```diff
@@ -140,3 +140,19 @@ customRating.addEventListener('value-changed', (event) =>  {
   document.getElementById('result').textContent = 'You have selected: ' + event.detail.value;
 });
 ```
+
+## Accessibility
+::a11y-intro::
+
+`ef-rating` does not assign any roles as it's not an actionable component.
+
+However, in the interactive mode, rating will be assigned `role="slider"`. Users can use arrow keys to update value. Current value of the rating will be updated to `aria-valuenow`. You can assign `aria-label` and `aria-valuetext` to `ef-rating` to give the component an assistive name for screen reader.
+
+If you need non-interactive rating to be focusable, you can set `tabindex="0"` and assigning `aria-label` value.
```

Comment:
This does not make sense. Non-interactive should not be focusable.

Reply:
updated